src/GrpcProtos/capture.proto

```diff
@@ -139,6 +139,150 @@ message ApiEvent {
   fixed64 r5 = 9;
 }
 
+message ApiScopeStart {
+  int32 pid = 1;
+  int32 tid = 2;
+  uint64 timestamp_ns = 3;
+
+  fixed64 encoded_name_01 = 4;
+  fixed64 encoded_name_02 = 5;
+  fixed64 encoded_name_03 = 6;
+  fixed64 encoded_name_04 = 7;
+  fixed64 encoded_name_05 = 8;
+  fixed64 encoded_name_06 = 9;
+  fixed64 encoded_name_07 = 10;
+  fixed64 encoded_name_08 = 11;
+  fixed64 encoded_name_09 = 12;
+  fixed64 encoded_name_10 = 13;
+  fixed64 encoded_name_11 = 14;
+  fixed64 encoded_name_12 = 15;
+  fixed64 encoded_name_13 = 16;
+  fixed64 encoded_name_14 = 17;
+  fixed64 encoded_name_15 = 18;
+  fixed64 encoded_name_16 = 19;
```

Comment:
so this takes us to 128 characters.. isn't this too much considering we have additional_encoded_name?

Reply:
Went down to 64. Still think we should have more than the original 34.